src/main/java/uk/gov/ons/ssdc/caseprocessor/model/dto/SkippedMessage.java

```diff
@@ -8,8 +8,7 @@
   private String messageHash;
   private byte[] messagePayload;
   private String service;
-  private String queue;
-  private String routingKey;
```

Comment:
shouldn't have removed `routingKey`... that will end up being renamed to `topic`